pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -188,7 +188,7 @@ func NewKubeGenericRuntimeManager(
 		logReduction:        logreduction.NewLogReduction(identicalErrorDelay),
 	}
 
-	typedVersion, err := kubeRuntimeManager.runtimeService.Version(kubeRuntimeAPIVersion)
+	typedVersion, err := kubeRuntimeManager.getTypedVersion()
```

Comment:
As long as you're making this change, can you also fix line 264 to call `getTypedVersion()` too?

Also, that will cause double-logging, since that function logs the error and also returns it (which shouldn't be done), so can you also fix those functions to skip logging the error and just add the context to the return value?

Reply:
Sure, let me handle it

Reply:
@tallclair code updated, PTAL